src/main/java/uk/gov/digital/ho/hocs/info/domain/model/Field.java

```diff
@@ -4,6 +4,7 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import uk.gov.digital.ho.hocs.info.api.dto.FieldDto;
```

Comment:
I don't think this import is used is it?

Reply:
I've removed this.